vendor/google.golang.org/grpc/stream.go

```diff
@@ -149,7 +149,7 @@ type clientStream struct {
 
 	tracing bool // set to EnableTracing when the clientStream is created.
 
-	mu sync.Mutex
```

Comment:
Please do not modify folders under the vendor path. If you run `make linter` it will run `goimports` which runs `go fmt` and the /vendor directory will be excluded.

These will need to be removed before the PR can be merged. If we do ever want to modify files under the vendor folder, we will want to make PRs in their respective repositories.